{
	"about": "https://jvndb.jvn.jp/ja/contents/2021/JVNDB-2021-004961.html",
	"title": "BusyBox における例外的な状態の処理に関する脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2021/JVNDB-2021-004961.html",
	"description": "BusyBox には、例外的な状態の処理に関する脆弱性が存在します。",
	"identifier": "JVNDB-2021-004961",
	"references": [
		{
			"text": "https://jvn.jp/vu/JVNVU99464755/index.html",
			"source": "JVN",
			"id": "JVNVU#99464755"
		},
		{
			"text": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-28831",
			"source": "CVE",
			"id": "CVE-2021-28831"
		},
		{
			"text": "https://nvd.nist.gov/vuln/detail/CVE-2021-28831",
			"source": "NVD",
			"id": "CVE-2021-28831"
		},
		{
			"text": "https://www.cisa.gov/news-events/ics-advisories/icsa-23-166-10",
			"source": "ICS-CERT ADVISORY",
			"id": "ICSA-23-166-10"
		},
		{
			"text": "https://cwe.mitre.org/data/definitions/755.html",
			"id": "CWE-755",
			"title": "例外的な状態における不適切な処理(CWE-755)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:busybox:busybox",
			"version": "2.2",
			"vendor": "BusyBox",
			"product": "BusyBox"
		},
		{
			"text": "cpe:/o:debian:debian_linux",
			"version": "2.2",
			"vendor": "Debian",
			"product": "Debian GNU/Linux"
		},
		{
			"text": "cpe:/o:fedoraproject:fedora",
			"version": "2.2",
			"vendor": "Fedora Project",
			"product": "Fedora"
		}
	],
	"cvss": [
		{
			"version": "2.0",
			"score": "5.0",
			"type": "Base",
			"severity": "Medium",
			"vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
		},
		{
			"version": "3.0",
			"score": "7.5",
			"type": "Base",
			"severity": "High",
			"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"date": "2023-06-16T14:23+09:00",
	"issued": "2021-12-02T18:08+09:00",
	"modified": "2023-06-16T14:23+09:00"
}
